packages/applications-service-api/src/repositories/representation.backoffice.repository.js

```diff
@@ -89,15 +88,20 @@ const getRepresentations = async (options) => {
 };
 
 const getFilters = async (caseReference) => {
-	const sql = Prisma.sql`
-      SELECT DISTINCT(representationType), status, count(id) as total
-      FROM Representation
-      WHERE caseReference = ${caseReference}
-        AND (status = 'PUBLISHED' or status = 'published')
-        AND representationType is not null
-      GROUP BY representationType, status`;
+	const options = await prismaClient.representation.groupBy({
+		by: ['representationType'],
```

Comment:
Do we definitely want to omit the Group By status?

Reply:
We weren't using status in the counts so I thought we did not need it - happy to add it though. Although as I am testing this, I am seeing issues with the count so need to make some changes.